### PR TITLE
improve the app UI (navigation tabs) and fix typos

### PR DESCRIPTION
**Pull request by vertxt**
image

- styling for navigation tabs
- fixed typos, renamed variables
- changed the screenOrientation to portrait to prevent breaking the app layout when the device is rotated to a horizontal position.

### CODE REVIEW

**Comment by maqmatic** [comment]
I got this error message from Metro:
```
LOG  Running "calculator" with {"rootTag":51}
ERROR  The action 'NAVIGATE' with payload {"name":"Converter"} was not handled by any navigator.

Do you have a screen named 'Converter'?

If you're trying to navigate to a screen in a nested navigator, see https://reactnavigation.org/docs/nesting-navigators#navigating-to-a-screen-in-a-nested-navigator.

This is a development-only warning and won't be shown in production.
```

image

No idea what might have caused this. I can still see the layout and click on the tabs though.

File: src/App.tsx [hunk 1]
```diff
@@ -12,28 +13,29 @@ const Stack = createNativeStackNavigator();
...
 };
 
 type CalculatorAppProps = {
   navigation: NativeStackNavigationProp<RootStackParamList, 'CalculatorApp'>;
 };
 
 type ConverterProps = {
-  navigation: NativeStackNavigationProp<RootStackParamList, 'Converter'>;
+  navigation: NativeStackNavigationProp<RootStackParamList, 'ConverterApp'>;
 };
```
Comment (maqmatic) [inline src/App.tsx:26]: `CalculatorAppProps` and `ConverterProps` are declared here but they're not used.

**Comment by vertxt** [comment]
I have resolved this problem, u can try again.

**Review by maqmatic — APPROVED** [review]
OK!

**Pull request merged**